src/api/app/views/webui2/webui/project/_pulse_list_builds.html.haml

```diff
@@ -0,0 +1,23 @@
+- if @builds.any?
+  - builds = @builds.group(:event_type).count
+  There
+  - # FIXME: 'has' does not have an inflection and
+  - # 'has'.pluralize(builds.values.sum, plural: 'have')
```

Comment:
because pluralize doesn't accept a param `plural`: https://api.rubyonrails.org/classes/String.html#method-i-pluralize What you are trying to do is not doable, please remove this comment.

Reply:
It's possible with rails pluralize :)